RayLin.md

```diff
@@ -19,6 +18,9 @@
 		- the appearance of small size video - Bug 1311700
 		- should show error state and error text should fit on most video size
 		- top level synthetic media - buttons such as play, pause, mute, CC, fullscreen should work - Bug 1312334
+  - Fennec
+    - should play video when click on video element
+    - should adjust all the controls' size accordingly
```

Comment:
I'll update this section soon later.

First regression is filed by Alastor which addresses similar issue.
I pointed out the second issue in one of the comment of Bug 1271765, and I'm gonna file this bug now.